e2e/test/AzureSecurityCenterForIoTLogAnalyticsClient.cs

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+#if NET451
```

Comment:
New features should not be developed for NET451. Please remove 451 support replacing it with `PlatformNotSupportedException`.